src/index.ts

```diff
@@ -623,6 +623,9 @@ export class Bigtable {
     const callback =
       typeof optionsOrCallback === 'function' ? optionsOrCallback : cb;
 
+    if (arrify(options.clusters).some(cluster => !cluster.id)) {
```

Comment:
Could you move this check/throw down to inside of the `reduce`? We're already arrifying and iterating the clusters down there, so we might as well keep it together. I have an example below, and I made a small change to the wording of the error. Sorry for not noticing sooner!

```js
reqOpts.clusters = arrify(options.clusters!).reduce((clusters, cluster) => {
  if (!cluster.id) {
    throw new Error('A cluster was provided without an `id` property defined.');
  }

  clusters[cluster.id!] = {
    location: Cluster.getLo
```